src/Microsoft.CodeQuality.Analyzers/Core/ApiDesignGuidelines/CollectionsShouldImplementGenericInterface.cs

```diff
@@ -172,8 +172,8 @@ public override void Initialize(AnalysisContext analysisContext)
             context.ReportDiagnostic(Diagnostic.Create(Rule,
                                                        namedTypeSymbol.Locations.First(),
                                                        namedTypeSymbol.Name,
-                                                       implementedInterface.ToDisplayString(SymbolDisplayFormat.MinimallyQualifiedFormat),
-                                                       missingInterface.ToDisplayString(SymbolDisplayFormat.MinimallyQualifiedFormat)));
+                                                       implementedInterface!.ToDisplayString(SymbolDisplayFormat.MinimallyQualifiedFormat),
+                                                       missingInterface!.ToDisplayString(SymbolDisplayFormat.MinimallyQualifiedFormat)));
```

Comment:
💡 Replace the `Debug.Assert` with `RoslynDebug.Assert` and these suppressions are not necessary.